data/jobs.txt

```diff
@@ -1573,3 +1573,192 @@ mission "Large Rush Delivery [3]"
 		payment
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Dump garbage"
```

Comment:
bit of a plot hole as to why you can't dispose garbage by dumping it into a nearby star :)